### DISCUSSION TITLE
Transaction Title Doubled

### DISCUSSION THREAD

**Discussion by Cryptec** [firefly-iii/firefly-iii#9191] [General]
Hi everyone, 

since a view Weeks all my transaction names from recurring transactions (with bills?) are doubled like name:name, which makes the overview look like a mess because of long names. is there a reason why or a solution? Thanks in advance. 

![image](https://github.com/user-attachments/assets/19780f1b-2f56-4e80-a5ac-41dd0854fc61)

regards

**Comment by JC5** [comment]
That could be a bug, but I can't be sure. I will check this.

What you can do to fix this:

- Edit the transaction. 
- Add a split using the "add a split" button
- Scroll down, and remove the "group title" (make the field empty)
- Remove the newly added split again using the button.
- Save the transaction.

**Reply by Cryptec** [reply]
thanks, your fix is working!

**Comment by JC5** [comment]
See also https://github.com/firefly-iii/firefly-iii/issues/9305
